.github/workflows/ci.yml

```diff
@@ -3,12 +3,12 @@ name: Python CI
 on:
   push:
     branches: [ master ]
-  pull_request:
+  pull_request_target:
```

Comment:
Would rather this run in a different workflow than changing the CI workflow

Reply:
we're going to to need the pull_request target anyway at some point, I thought of creating another test.yml, but then the ci.myl only had the lints and it was awkward.